### PR TITLE
 ypersly - multi snapshot csi times

### PR DESCRIPTION
**Pull request by ypersky1980**
This PR handles the following: 

1) Adds calculation and reporting ( including uploading to the ES) of CSI measurements for multiple snapshot creation.

2) Changes calculation of Total multiple snapshot creation time ( so that the same method as in the single snapshot test will be used)

3) Removes redundant parts of the code 

4) Fixes teardown problem - no need to try to delete instances ( pvc object ) after Results test . 

Signed-off-by: Yulia Persky <ypersky@redhat.com>

### CODE REVIEW

File: tests/e2e/performance/csi_tests/test_pvc_multi_snapshot_performance.py [hunk 1]
```diff
@@ -340,7 +317,7 @@ def create_snapshot(self, snap_num):
         """
         log.info(f"Taking snapshot number {snap_num}")
         # Getting UTC time before test starting for log retrieve
-        UTC_datetime = datetime.datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%SZ")
+        start_time = datetime.datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%SZ")
```
Comment (Avilir) [inline tests/e2e/performance/csi_tests/test_pvc_multi_snapshot_performance.py:320]: can be replace with 
`start_time = self.get_time("csi")`

Comment (ypersky1980) [reply tests/e2e/performance/csi_tests/test_pvc_multi_snapshot_performance.py]: Done, thank you.

**Review by yati1998 — COMMENTED** [review]
LGTM

**Review by Avilir — APPROVED** [review]
LGTM

**Comment by ypersky1980** [comment]
> PR validation
> 
> Cluster Name: Cluster Configuration: PR Test Suite: performance PR Test Path: tests/e2e/performance/csi_tests/test_pvc_multi_snapshot_performance.py Additional Test Params: OCP VERSION: 4.10-ga OCS VERSION: 4.10 tested against branch: master
> 
> [Job UNSTABLE](https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-trigger-test-pr/2830//testReport) (some or all tests failed).


It's validated since the test reported ERROR and not failure. The test finished properly and passed.

**Pull request merged**